UITests/TransferMethod/ListTransferMethodTests.swift

```diff
@@ -323,7 +323,7 @@ class ListTransferMethodTests: BaseTests {
     }
 
     func verifyRemoveConfirmation(transferMethod: String) {
-          let confirmMessage = listTransferMethod.getConfirmationMessage(transferMethod: transferMethod)
+          let confirmMessage = listTransferMethod.removeAccountMessage
```

Comment:
Delete function `listTransferMethod.getConfirmationMessage(transferMethod: transferMethod)` if it's not required anymore.